src/features/profile/components/Header/NonBeneficiaryHeader/NonBeneficiaryHeader.native.test.tsx

```diff
@@ -100,30 +91,33 @@ describe('<NonBeneficiaryHeader/>', () => {
       getByTestId('eligibility-banner-container')
     })
 
-    it('should display correct depositAmount', () => {
-      const { queryByText } = render(
-        <NonBeneficiaryHeader
-          eligibilityStartDatetime="2021-02-30T00:00Z"
-          eligibilityEndDatetime="2022-02-30T00:00Z"
-        />
-      )
-
-      expect(queryByText(/Profite de 300€/)).toBeTruthy()
-    })
-
-    it('should display correct credit message for underage', () => {
-      mockedUseIsUserUnderage.mockReturnValueOnce(true)
-
-      const { queryByText } = render(
-        <NonBeneficiaryHeader
-          eligibilityStartDatetime="2021-02-30T00:00Z"
-          eligibilityEndDatetime="2022-02-30T00:00Z"
-        />
-      )
+    it.each`
+      birthdate       | credit   | age
+      ${'2007-11-06'} | ${'20'}  | ${15}
+      ${'2006-11-06'} | ${'30'}  | ${16}
+      ${'2005-11-06'} | ${'30'}  | ${17}
+      ${'2004-11-06'} | ${'300'} | ${18}
+    `(
+      "should display a banner if the user has not finished the identification flow yet (user's age: $age)",
+      ({ birthdate, credit }: { birthdate: string; credit: string }) => {
+        mockdate.set(new Date('2022-12-01T00:00:00Z'))
```

Comment:
Les mockdates dans les tests me font peur en général car ce sont des mocks généraux : ici ça pose pas problème car les autres tests n'utilisent pas la date mais est-ce que tu penses que de futurs tests ici pourront ? 
Si oui, il vaut mieux unmock (si possible ?) ou remocker avec la valeur par défaut ?

Reply:
Si justement plein d'autres tests utilisent les mockset. Après je te rejoins sur le fait que c'est pas dingue d'avoir plein de mockset avec des valeurs différentes dans les it ...

Reply:
J'ai fait un peu de tri et il n'y a plus qu'un seul mockdate pour tout le fichier :)